tests.py

```diff
@@ -4,6 +4,30 @@ def test_add():
     pass
 
 def test_remove():
+
+    try:
+        print("Trying to remove")
+        data = {"one": 1, "two": 2}
+        test = Storage(data)
+        test.remove("one")
+    except KeyError as e:
+        print(e)
+        print("Remove function failed with data:", data)
+    else:
+        print("OK")
+
+    
+    try:
+        print("Trying to remove twice...")
+        data = {"one" : 1, "two" : 2}
+        test.remove("one")
+        test.remove("one")
```

Comment:
It's a good idea to follow rule that each test function should cover only one test-case: 'remove existed key' OR 'remove absent key'.

In line 11 you used dictionary _data_ to assign variable _test_. In python mutable variables are passed by reference that's why key "one" exists in variable _test_ after changing value of _data_ in line 22. 

But you can't be 100% sure that your _test_ variable is valid after executing first block (lines 8-17): some links can be crashed or smth else. May be it's better to implement different functions to test different behavior.